### PR TITLE
Added SparseCategoricalCrossEntropy Loss Function

### LINKED ISSUE #9: Add more loss functions like cross-entropy...
Right we have implemented only Mean Squared Error and Logarithmic Error as our loss functions in our module. As we are going to implement more and more machine learning algorithms we will need more loss functions. There in this issue you can add one or more loss functions like cross-entropy etc.

### PR DESCRIPTION
**Pull request by quadri-haider-ali**
Added Sparse Categorical Cross Entropy Loss Function
Fixes #9 

This pull request adds `SparseCategoricalCrossEntropy` Loss function in MLlib/loss_func.py

### CODE REVIEW

File: MLlib/loss_func.py [hunk 1]
```diff
@@ -38,3 +38,15 @@ def derivative(X,Y,W):
 		AbsError=(np.dot(X,W).T-Y)
 		return np.dot(np.divide(AbsError,np.absolute(AbsError),out=np.zeros_like(AbsError), where=(np.absolute(AbsError))!=0),X).T/M
 
+class SparseCategoricalCrossEntropy():
+    @staticmethod        
+    def loss(X, Y, W):
+        M=X.shape[0]
+        H=sigmoid(np.dot(X,W).T)*(len(Y)-1)
```
Comment (agrawalshubham01) [inline MLlib/loss_func.py:45]: use `softmax` and use input `one hot encoder` and also use `assert` to check if the given `y` is one hot encoder.

**Comment by TarunTomar122** [comment]
Closed due to Inactivity!
